packages/host/app/lib/stack-item.ts

```diff
@@ -149,3 +149,8 @@ export class StackItem {
     });
   }
 }
+
+export function isIndexCard(stackItem: StackItem) {
+  let realmURL = stackItem.card[stackItem.api.realmURL];
```

Comment:
if you merge this branch into this PR https://github.com/cardstack/boxel/pull/1694, `stackItem` as a new property `realmURL`, which would be easier to use..

Reply:
actually sorry, nevermind, I'm going in a different direction with that PR. what you have is just fine.

Reply:
no problem!